<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuildProcessStartDialog</class>
 <widget class="QDialog" name="BuildProcessStartDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select files</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="Headline">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Select output files</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Information">
           <property name="text">
            <string>Selected files will be safed in the ouptut directory after the build</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Checkboxes">
         <item>
          <layout class="QVBoxLayout" name="CheckboxLeft">
           <property name="spacing">
            <number>1</number>
           </property>
           <item>
            <widget class="QCheckBox" name="sourceCheck">
             <property name="toolTip">
              <string>Current source.xml of the project</string>
             </property>
             <property name="text">
              <string>source.xml</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="ImageCheck">
             <property name="toolTip">
              <string>Image</string>
             </property>
             <property name="text">
              <string>Image</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="LicenseCheck">
             <property name="toolTip">
              <string>License file</string>
             </property>
             <property name="text">
              <string>licence.txt</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="LicenseXmlCheck">
             <property name="toolTip">
              <string>xml License file</string>
             </property>
             <property name="text">
              <string>licence.xml</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="CheckboxRight">
           <property name="spacing">
            <number>1</number>
           </property>
           <item>
            <widget class="QCheckBox" name="ValidationCheck">
             <property name="toolTip">
              <string>Package list validation result</string>
             </property>
             <property name="text">
              <string>validation.txt</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="ReportCheck">
             <property name="toolTip">
              <string>Report</string>
             </property>
             <property name="text">
              <string>elbe-report.txt</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="LogCheck">
             <property name="toolTip">
              <string>Log file</string>
             </property>
             <property name="text">
              <string>log.txt</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="Information_2">
         <property name="text">
          <string>Do you want to build the sources-/bin-CDROM?</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="binCdCheck">
           <property name="text">
            <string>src-cdrom.iso</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="sourceCdCheck">
           <property name="text">
            <string>bin-cdrom.iso</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BuildProcessStartDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BuildProcessStartDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
